charts/tidb-operator/values.yaml

```diff
@@ -15,11 +15,28 @@ timezone: UTC
 operatorImage: pingcap/tidb-operator:v1.1.0-beta.1
 imagePullPolicy: IfNotPresent
 
-defaultStorageClassName: local-storage
+#
+# Specify storage class for pd/tikv componnents. 
+#
+# By default, "standard" is used. If you want to use default storage class in
+# the Kubernetes cluster, set its value to "".
+#
+# https://kubernetes.io/docs/tasks/administer-cluster/change-default-storage-class/
+# 
+# defaultStorageClassName: standard 
```

Comment:
- use user-supplied class if provided
- use tidb-operator default class if not-empty (defaults to "standard" for backward compatibility)
- use Kubernetes cluster default class

Reply:
if we want to use the default storage class of Kubernetes cluster **by default**, we need to empty the default values of `--default-storage-class-name` and `--default-backup-storage-class-name`.

cc @aylei what do you think?

Reply:
Yes, I think so

Reply:
Looks like that this change is not backward compatible.

Thinks a user upgrade operator with defaultStorageClassName empty, the storageClass of current PVC and PVCTemplate will be synced to the cluster default, which breaks reconciliation because storageClassName is immutable IIRC